package.json

```diff
@@ -24,7 +24,8 @@
   "homepage": "https://github.com/great-lakes/project-egypt-generators#readme",
   "dependencies": {
     "g.js": "^0.1.14",
-    "npm": "^6.3.0"
+    "npm": "^6.3.0",
+    "yarn": "^1.9.4"
```

Comment:
you don't need npm or yarn for dependencies.  please remove both.

Reply:
changes have been made in latest commit